packages/components/src/InputFile/InputFile.tsx

```diff
@@ -84,7 +84,7 @@ interface InputFileProps {
    * Allowed File types.
    * @default "all"
    */
```

Comment:
@JordanLos sorry, one last thing

```suggestion
   * Allowed File types.
   * 
   * @param "images" - only accepts all types of image
   * @param "basicImages" - only accepts png, jpg and jpeg
   *
   * @default "all"
   */
```